radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -172,23 +172,22 @@ impl FeeTable {
     #[inline]
     pub fn before_invoke_cost(&self, _actor: &Actor, input_size: usize) -> u32 {
         add(
-            13 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+            10 / CPU_INSTRUCTIONS_TO_COST_UNIT,
             Self::data_processing_cost(input_size),
         )
     }
 
     #[inline]
     pub fn after_invoke_cost(&self, input_size: usize) -> u32 {
-        // used max cpu instruction counts
         add(
-            4756 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+            6 / CPU_INSTRUCTIONS_TO_COST_UNIT,
```

Comment:
This equals 0, should we just change this to 1?

Reply:
Which is the `6` derived from?